### DISCUSSION TITLE
Showcase

### DISCUSSION THREAD

**Discussion by obritoluis** [flyntwp/flynt#380] [Ideas]
Hi Flynt team,

Congratulations on this project! 🙌
We love Flynt ❤️ 
Flynt + Timber ignited our love for WordPress again 🔥 😀

I was wondering if you wanted to use some of the websites that we built using Flynt, to showcase to the world. 🙂

Here they are:
https://www.15five.com/
https://practicepanther.com/
https://aventigroup.com/
https://www.goworkspace.com/

**Comment by aaronmeder** [comment]
Cool Idea! I +1 it 👍🏻 

It could be beneficial for multiple reasons:
- Agencies / web workers could present themselves as being "Flynt" capable partners
- A showcase will show that Flynt gets real work done :)

Our sites so far:
- [https://thinkproject.com](https://thinkproject.com)
- [https://franziskaaeschimann.ch](https://franziskaaeschimann.ch)
...and a few more we are working on right now.

**Comment by timohubois** [comment]
@obritoluis and @aaronmeder we are internally thinking about a more elegant way to share custom made components. What would you like to see? And what would you expect from this feature?
